optuna/integration/keras.py

```diff
@@ -1,6 +1,8 @@
 from typing import Dict
 from typing import Optional
 
+import warnings
+
```

Comment:
Please remove the blank line since `warnings` is a standard library.
```suggestion
import warnings

```